tests/ui/README.md

```diff
@@ -25,7 +25,7 @@ The tests must be run in Firefox 48 or later.
 To run against a different environment, pass in a value for `--base-url`, like so:
 
 ```bash
-$ tox -e discopane-ui-tests -- --base-url https://discovery.addons.mozilla.org
+PYTEST_ADDOPTS="--base-url https://discovery.addons.mozilla.org" tox -e discopane-ui-tests
```

Comment:
Ah, my guess is that the space between `--base-url` and `https://discovery.addons.mozilla.org` is causing path confusion when locating `setup.cfg`. You could also try replacing the space with an `=`.